### ISSUE TITLE
Undefined subroutine &Crypt::Argon2::argon2_needs_rehash called 

### ISSUE THREAD

**Issue by eserte** [Leont/crypt-passphrase-argon2#1]
t/basic.t fails on some of my smokers, probably if the installed Crypt::Argon2 is too old:
```
Undefined subroutine &Crypt::Argon2::argon2_needs_rehash called at /usr/home/cpansand/.cpan/build/2021030807/Crypt-Passphrase-Argon2-0.001-2/blib/lib/Crypt/Passphrase/Argon2.pm line 40.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/basic.t ....... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 1 subtests passed 
```

Statistical analysis:
```
****************************************************************
Regression 'mod:Crypt::Argon2'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   0.74
[1='eq_0.005'] 	     -0.0000	      0.0000	  -0.52
[2='eq_0.006'] 	     -0.0000	      0.0000	  -1.04
[3='eq_0.007'] 	     -0.0000	      0.0000	  -2.13
[4='eq_0.008'] 	      1.0000	      0.0000	6249112852107506.00
[5='eq_0.009'] 	      1.0000	      0.0000	6572718151853355.00

R^2= 1.000, N= 83, K= 6
****************************************************************
```

**Comment by eserte** [comment]
Another possible test output:
```
"argon2id_pass" is not exported by the Crypt::Argon2 module
Can't continue after import errors at t/basic.t line 8.
BEGIN failed--compilation aborted at t/basic.t line 8.
t/basic.t ....... 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by Leont** [comment]
Thank you. Yes I should have explicitly depended on the latest version

**Comment by Leont** [comment]
This has been fixed in 0.002 :-)
